src/components/popup-menu/popup-menu.scss

```diff
@@ -0,0 +1,38 @@
+$userPanelBackground: $blueSecondary;
+$userPanelPadding: $layoutDefaultPadding;
+
+$include-html: false !default;
+
+@if ($include-html) {
+  .mint-popup-menu {
+    position: relative;
+    width: 100%;
+    padding: $userPanelPadding/3 $userPanelPadding/2;
```

Comment:
> Why width:100% is not width: $userPanelWidth

hm, good point - I treat `width:100%` as an equivalent of saying "take all horizontal space". I don't expect anyone to change value "100%", either this line exists or not. That's why I haven't created a `$userPanelWidth ` variable.

> but padding:$userPanelPadding/3 and not padding: $layoutDefaultPadding/3

`$userPanelPadding` exists because I'm using it in multiple places in that component, if someone decides to change its value to `$userPanelPadding: layoutDefaultPadding / 2` it will change all dependent paddings proportionally.